app/components/works/title_component.html.erb

```diff
@@ -12,5 +12,5 @@
      </div>
    </div>
 
-  <%= render Works::ContactEmailComponent.new(form: form) %>
+  <%= render Works::ContactEmailComponent.new(form: form, key: 'work.contact_email') %>
```

Comment:
I'm not seeing that i18n key in `config/locales/en.yml`. Did I miss it?

Reply:
It's not there yet (different ticket), but this component requires a key. It gracefully handles missing translations.